pep-0669.rst

```diff
@@ -186,13 +245,18 @@ Callback function arguments
 When an active event occurs, the registered callback function is called.
 Different events will provide the callback function with different arguments, as follows:
 
-* All events starting with ``PY_``:
+* ``PY_START`` and ``PY_RESUME``:
 
     ``func(code: CodeType, instruction_offset: int) -> DISABLE | Any``
```

Comment:
NB, all of these these would likely display better as code blocks (proper breaks, syntax highlighting, background, more easily copyable, no quote formatting/italics, etc) and with less formatting too. For example:

```suggestion
* ``PY_START`` and ``PY_RESUME``::

    func(code: CodeType, instruction_offset: int) -> DISABLE | Any
```